control/modelsimp.py

```diff
@@ -211,26 +218,46 @@ def balred(sys, orders, method='truncate'):
         of systems)
     method: string
         Method of removing states, either ``'truncate'`` or ``'matchdc'``.
+    alpha: float
+        Specifies the alpha-stability boundary for the eigenvalues
```

Comment:
If the text was indeed copied, can you create a new description of the `alpha` parameter? An easy solution for now is to write a brief summary and indicate where more information can be found (the documentation of SLICOT).